common/src/main/kotlin/io/goooler/demoapp/common/base/binding/BaseBindingFragment.kt

```diff
@@ -30,4 +32,9 @@ abstract class BaseBindingFragment<VB : ViewDataBinding> :
     super.onViewCreated(view, savedInstanceState)
     binding.lifecycleOwner = viewLifecycleOwner
   }
+
+  override fun onDestroyView() {
+    _binding = null
+    super.onDestroyView()
+  }
```

Comment:
Release `binding`'s reference when current fragment was attached to parent with `replaceFragment(isAddToBackStack = true)`

Reply:
The previous fragment was called `onDestroyView()` but not `onDestroy()`, LeakCanary analyzes the view held by fragment should be released.

Reply:
https://github.com/square/leakcanary/blob/18f364bcbe14fc31547cb0c18e52414da999370f/leakcanary-object-watcher-android-androidx/src/main/java/leakcanary/internal/AndroidXFragmentDestroyWatcher.kt#L39-L50